packages/anoncreds/src/utils/w3cAnonCredsUtils.ts

```diff
@@ -166,6 +167,14 @@ export function getStoreCredentialOptions(
   return storeCredentialOptions
 }
 
+// The issuer of the schema does not always match the issuer of the credential definition thus the unqualified schema id needs to be derived from both values
+function getUnqualifiedSchemaId(schemaIssuerId: string, schemaId: string) {
+  const schemaDid = schemaIssuerId.split(':')[3]
+  const split = getUnQualifiedDidIndyDid(schemaId).split(':')
+  split[0] = schemaDid
+  return split.join(':')
+}
+
```

Comment:
I made a comment in the discord about this but it looks like Credo is not reconstructing the schema id correctly in some cases. I have found that his will work but I cannot track if it is a issuer higher up in the code or not. I would be happy to fix it higher up if anyone see how to.